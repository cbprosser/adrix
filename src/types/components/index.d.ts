export * from './AdvancedSearchForm';
export * from './MTGStaticTypes';
export * from './MultipleCardDisplay';
export * from './SimpleCardSearchBar';
export * from './Symbol';
