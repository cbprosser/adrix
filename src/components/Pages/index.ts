export * from './AdvancedSearch';
export * from './Home';
export * from './Results';
