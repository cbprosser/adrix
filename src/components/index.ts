export * from './AppHeader';
export * from './CardDisplay';
export * from './Pages';
export * from './SimpleCardSearchBar';
