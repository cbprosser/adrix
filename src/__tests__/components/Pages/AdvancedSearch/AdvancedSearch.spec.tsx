import { render } from '@testing-library/react';
import { AdvancedSearch } from '../../../../components';

jest.mock('../../../../components/AdvancedSearchForm', () => ({
  AdvancedSearchForm: () => <div>AdvancedSearchForm Mocked</div>,
}));

describe('AdvancedSearch', () => {
  it('should match the snapshot', () => {
    const { container } = render(<AdvancedSearch />);
    expect(container).toMatchSnapshot();
  });
});
