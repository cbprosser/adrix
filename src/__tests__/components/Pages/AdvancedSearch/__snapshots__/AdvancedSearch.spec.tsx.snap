// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvancedSearch should match the snapshot 1`] = `
<div>
  <div>
    AdvancedSearchForm Mocked
  </div>
</div>
`;
